packages/components/Tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,152 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text } from '@fluentui-react-native/text';
+import { FocusZone } from '@fluentui-react-native/focus-zone';
+import { tabsName, TabsType, TabsProps, TabsState, TabsSlotProps, TabsRenderData, ITabsContext } from './Tabs.types';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
```

Comment:
Not blocking, but it's worth noting that this control is made with the legacy framework, rather than the newer compose,  which I will try to stop calling experimental since I don't think it is at this point.